### ISSUE TITLE
GPU Not working again

### ISSUE THREAD

**Issue by gitihobo** [JonathanFly/bark#83]
Reinstalled Bark infinity, simply git cloned and installed requirements-pip.txt but GPU no longer loads, what do we install?

**Comment by JonathanFly** [comment]
I'll be around tomorrow night in the official Bark discord, but try for now, it's what I used to to install music gen in the same as conda:

1) https://github.com/JonathanFly/bark/commit/738f20f570e4531f42efa1ddac85af31d1f28219

or 

2) https://github.com/JonathanFly

![m_small](https://github.com/JonathanFly/bark/assets/163408/e4453ee3-3d7a-4264-bd05-51682592c9cb)

Been too distracted...

**Comment by gitihobo** [comment]
I am really confused now, I am new to this, all I did was git clone this repo and install requirements-pip.txt inside my real anaconda venv, It never installed Mamba, all I do is enter python bark_webui.py to see the webui, I now installed musicgen-conda-pip.txt to see if I could fix the no GPU issue for BARK AND try out MusicGen at the same time, I do not see GPU fixed in Bark and I don't see a musicgen_webui.py file

**Comment by JonathanFly** [comment]
Updating shortly.

**Comment by gitihobo** [comment]
Awesome thank you Jon

**Comment by JonathanFly** [comment]
> Awesome thank you Jon

I'm fixing the one click installer, so you may need some help if you installed it a different way. But I'll be in Bark discord for new few hours.

**Comment by gitihobo** [comment]
I don't use discord, can you tell us the basics here? I'm guessing it's even easier than my method.

**Comment by JonathanFly** [comment]
> 

Try the zip file method: https://github.com/JonathanFly/bark#install-steps

I'll be online for a few hours, let me know if anything isn't working and I can fix.

**Comment by gitihobo** [comment]
Ok so for orders sake I made another anaconda enviroment, put the extracted zip file and launched install, I got this one-click-bark-installer_june16_23\installer_files\env" >nul && conda install -y -k ffmpeg-downloader --solver=libmamba' failed with exit status code '2'. Exiting...Press any key to continue . . .

**Comment by JonathanFly** [comment]
> Ok so for orders sake I made another anaconda enviroment, put the extracted zip file and launched install, I got this one-click-bark-installer_june16_23\installer_files\env" >nul && conda install -y -k ffmpeg-downloader --solver=libmamba' failed with exit status code '2'. Exiting...Press any key to continue . . .

I put the wrong files in the zip, just fixed it. Way too sleepy tonight...

**Comment by gitihobo** [comment]
I will try it out now, we don't blame you, we appreciate you and we depend on you

**Comment by JonathanFly** [comment]
BTW if you do get it installed, start the `TROUBLESHOOT_bark_setup_manually_by_entering_the_conda_environment.bat` file and type `conda clean --all` and it should clear up a bunch of disk space from the failed installations with file that are no longer needed.

**Comment by gitihobo** [comment]
So I have a couple of questions, first thing is the script seems to turn my directory path into a virtual environment, does that mean I dont need to make the environment myself? I would prefer to since that way I can have all my envs in anaconda>envs but anyway, this is the error I seem to get conda-script.py install: error: argument --solver: invalid choice: 'libmamba' (choose from 'classic')
Command  nul && conda install -y -k pip --solver=libmamba' failed with exit status code '2'. Exiting...
Press any key to continue . . .

**Comment by JonathanFly** [comment]
> 

Yeah it installs basically a standalone conda. You don't need to make the environment tor even download the conda exe.

The problem you're getting is because I used the mamba fast solver in the version I downloaded. If you want to install it in your own ENV, you might have to wait for me to update regular install.

**Comment by mediashock** [comment]
> 

What is that MusicGen screenshot of?

**Comment by JonathanFly** [comment]
> > 
> 
> hey, What version is that MusicGen screenshot of?

I was working on a MusicGen UI but never finished it. I might just push the half working version though. It still has some cool featutres.
